### DISCUSSION TITLE
How to download mill source code

### DISCUSSION THREAD

**Discussion by romain-gilles-ultra** [com-lihaoyi/mill#2934] [Q&A]
Hi Mill,
I'm using the **Project-specific installation** mechanism (version: `0.11.6`) but it sounds like the source code of mill itself is not automatically available. I may not be looking at the right place.
Can you help me :pray: 

Thanks

**Comment by lefou** [comment]
What do you mean with that? Are you editing the build.sc in an IDE and try to jump to definition / API?
